linkerd/app/core/src/svc.rs

```diff
@@ -18,7 +18,11 @@ use tower::{
     make::MakeService,
 };
 pub use tower::{
-    layer::Layer, service_fn as mk, spawn_ready::SpawnReady, util::Either, Service, ServiceExt,
+    layer::Layer,
+    service_fn as mk,
+    spawn_ready::SpawnReady,
+    util::{Either, MapErrLayer},
```

Comment:
```suggestion
    util::{Either, MapErr},
```

Reply:
i actually tried this initially and it didn't compile — `MapErr` is generic over a Service and a fn, while `MapErrLayer` is only generic over the fn type. using the `layer` fn requires a type for the Service type param, which apparently can't be inferred in this usage. :/